README.md

```diff
@@ -0,0 +1,13 @@
+# pac-cmd
+
+A command line tool to change proxy auto-config settings of operation system. Binaries included. Simply `make` to build it.
+
+# Usage
+
+```sh
+pac [on  <pac url> | off]
+```
+
+*  **Mac**: Have to `sudo` as setting pac is an privileged action.
+
+*  **Windows**: Install [MinGW-W64](http://sourceforge.net/projects/mingw-w64) to build pac, as it has up to date SDK headers we require.
```

Comment:
What are the pros and cons of building with MinGW instead of Visual Studio?  Since we already require visual studio for some other Windows projects, does it make sense to just use Visual Studio for this too?